datafusion/core/src/datasource/listing/helpers.rs

```diff
@@ -336,6 +336,20 @@ fn batches_to_paths(batches: &[RecordBatch]) -> Result<Vec<PartitionedFile>> {
         .collect()
 }
 
+fn to_timestamp_millis(v: i64) -> Result<chrono::DateTime<Utc>> {
```

Comment:
Likewise, now if an error converting timestamps occurs, DataFusion will make a real error, not panic